versioned_docs/version-21.10/releases/centreon-commercial-extensions.md

```diff
@@ -118,6 +118,23 @@ Release date: `February 18, 2022`
 
 ## Centreon Auto Discovery
 
+### 21.10.3
+
+Release date: `null`
+
+#### Improvements
+
+- In the Host Discovery job result listing, the red "warning" sign that used to point out that a discovered host already existed has been replaced with a yellow one because it suggested that a severe problem occurred. The "info" sign that lead to the raw discovery data has been replaced with a "text file" icon.
```

Comment:
```suggestion
- In the Host Discovery job result listing, the red "warning" sign that used to point out that a discovered host already existed has been replaced with a yellow one because the colour red suggested that a severe problem had occurred. The "info" sign that led to the raw discovery data has been replaced with a "text file" icon.
```